sdks/go/opspec/interpreter/call/op/outputs/interpret.go

```diff
@@ -1,28 +1,73 @@
 package outputs
 
 import (
+	"fmt"
+	"sort"
+	"strings"
+
 	"github.com/opctl/opctl/sdks/go/model"
+	"github.com/opctl/opctl/sdks/go/opspec"
 	"github.com/opctl/opctl/sdks/go/opspec/interpreter/call/op/params"
 )
 
 // Interpret applies defaults to & validates output args
 func Interpret(
 	outputArgs map[string]*model.Value,
 	outputParams map[string]*model.Param,
+	callOutputs map[string]string,
 	opPath string,
 	opScratchDir string,
 ) (
 	map[string]*model.Value,
 	error,
 ) {
-
 	outputArgs, err := params.Coerce(outputArgs, outputParams, opScratchDir)
-	if nil != err {
+	if err != nil {
```

Comment:
I think we should do a mass replacement of this ordering. Some languages you can accidentally assign the thing you're comparing so putting the constant on the left keeps you from doing that accidentally. That's not a thing in golang and the ecosystem is more used to err on left so I think I should concede and go with the flow.

Reply:
https://github.com/opctl/opctl/pull/897

Reply:
I very much agree. I see warnings about this from my default linters sometimes (https://staticcheck.io/docs/checks#ST10170) and the `nil != err` pattern isn't something I've seen in any other go codebase.